hetio/abbreviation.py

```diff
@@ -37,13 +37,28 @@ def validate_abbreviations(metagraph):
         if not abbrev.isupper():
             print('lowercase metanode abbreviation:', abbrev)
             valid = False
+
+    # Check if metanode abbreviation starts with a digit
```

Comment:
I have a small preference that these checks are integrated into the for loop above. The existing comments like 

```python
     # Check capitalizations
     # metanode abbreviations should be uppercase
```

would be moved to the appropriate inner loop location.